Source/ui_win32/Res.rc

```diff
@@ -157,6 +169,8 @@ BEGIN
         MENUITEM "State Slot",                  ID_MAIN_VM_STATESLOT
         MENUITEM "Save State\tF7",              ID_MAIN_VM_SAVESTATE
         MENUITEM "Load State\tF8",              ID_MAIN_VM_LOADSTATE
+        MENUITEM SEPARATOR
```

Comment:
Yeah, VS has a bad habit of changing stuff in resource files when the users locale config don't match. It's better to filter that noise out whenever possible.